src/Apps/CollectorProfile/Routes/Saves2/Components/Actions/Mutations/useUpdateCollection.ts

```diff
@@ -0,0 +1,29 @@
+import { graphql } from "react-relay"
+import { useMutation } from "Utils/Hooks/useMutation"
+import { EditSavesModalMutation } from "__generated__/EditSavesModalMutation.graphql"
+
+export const useUpdateCollection = () => {
+  return useMutation<EditSavesModalMutation>({
+    mutation: graphql`
+      mutation EditSavesModalMutation($input: updateCollectionInput!) {
+        updateCollection(input: $input) {
+          responseOrError {
+            __typename
+            ... on UpdateCollectionSuccess {
+              collection {
+                internalID
+                name
+              }
+            }
```

Comment:
@dimatretyak I found that the the Artwork Lists rail gets an updated immediately thanks to the Relay store, as we discussed. Is that because the `name` is returned here by the mutation? 

(In other words: not an optimistic update, but an after-the-fact update based on the actual mutation result?)

Reply:
Yes, that's right